src/twitch-client.ts

```diff
@@ -309,12 +214,12 @@ export default class TwitchClient {
    *  console.log(response.users[0].display_name);
    * });
    */
-  public getTwitchUsers(usernames: string[]): Promise<TwitchUser[]> {
+  public getTwitchUsers(usernames: string[], jwt: string): Promise<TwitchUser[]> {
```

Comment:
Looks like `jwt` might be unused. Does it need to be passed to `signedTwitchHelixRequest`?